xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ if(CORE_SYSTEM_NAME STREQUAL android)
                       RendererMediaCodecSurface.h)
 endif()
 
-if(CORE_PLATFORM_NAME_LC STREQUAL gbm)
+if(CORE_PLATFORM_NAME_LC STREQUAL gbm AND OPENGLES_FOUND)
```

Comment:
`RendererDRMPRIME.cpp` should work with OpenGL not just OpenGLES but guess there is only GLES device/platforms that is using the direct-to-plane renderer at the moment

Reply:
If you don't mind I'd rather keep it as is until we really need it for OpenGL.